src/foam/nanos/ruler/predicate/MQL.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
I would like to handle this using the DOT notation (which we didn't do yet). Can you remove this from the PR?